lib/awesome_print/formatters/base_formatter.rb

```diff
@@ -123,18 +123,29 @@ def outdent
 
       def align(value, width)
         if options[:multiline]
```

Comment:
since you are here, it'd be great to resolve the ABCSize issues in this method. Return early `unless options[:multiline]`,

Reply:
ok, wasn't sure how far to take Metrics, and I attempted to clear it up some already.